### PR TITLE
Edits to CLI help text

### PR DESCRIPTION
**Pull request by joel-hamill**

### CODE REVIEW

File: cli/dcoscli/data/help/dcos.txt [hunk 1]
```diff
@@ -1,39 +1,42 @@
...
-
-'dcos help' lists all available subcommands. See 'dcos <command> --help'
-to read about a specific subcommand.
+Description:
+    The Mesosphere Datacenter Operating System (DCOS) spans all of the machines in
+your datacenter or cloud and treats them as a single, shared set of resources.
 
 Usage:
-    dcos [options] [<command>] [<args>...]
+     dcos [options] [<command>] [<args>...]
```
Comment (joel-hamill) [inline cli/dcoscli/data/help/dcos.txt]: Extra space.

File: cli/dcoscli/data/help/dcos.txt [hunk 2]
```diff
@@ -1,39 +1,42 @@
...
-                                [default: ~/.dcos/dcos.toml]
+Positional Arguments:
+    <log-level>
+        The severity level:
+        * debug    Prints all messages.
+        * info     Prints informational, warning, error, and critical messages.
+        * warning  Prints warning, error, and critical messages.
+        * error    Prints error and critical messages.
+        * critical Prints only critical messages to stderr.
+        This setting does not affect the output sent to stdout.
```
Comment (jsancio) [inline cli/dcoscli/data/help/dcos.txt]: This doesn't look right. I think we need to move this up to the `--log-level` entry in line 13.

Comment (jsancio) [inline cli/dcoscli/data/help/dcos.txt]: It is not a positional argument it is part of the `Options` above.

File: cli/dcoscli/data/help/help.txt [hunk 3]
```diff
@@ -1,11 +1,23 @@
-Display command line usage information
+Description:
+   Display help information about DCOS.
```
Comment (jsancio) [inline cli/dcoscli/data/help/help.txt]: Missing space.

File: cli/dcoscli/data/help/marathon.txt [hunk 4]
```diff
@@ -18,91 +19,120 @@ Usage:
...
+   group show
+       Print a detailed list of groups.
+   group remove
+       Remove a group.
+   group update
+       Update a group.
+   task list
+       List all tasks.
+   task show
+       List a specific task.
```
Comment (jsancio) [inline cli/dcoscli/data/help/marathon.txt]: Looks like we are missing a space for all of these entries.

Comment (jsancio) [inline cli/dcoscli/data/help/marathon.txt]: We will have to adjust the description once we adjust the command entry.

**Comment by jsancio** [comment]
First pass. Formatting comments only. I will go through it again.

File: cli/dcoscli/data/help/dcos.txt (hunk 2 above)
Comment (joel-hamill) [inline cli/dcoscli/data/help/dcos.txt]: Agreed @jgarcia-mesosphere

File: cli/tests/data/help/package.txt [hunk 5]
```diff
@@ -1,4 +1,5 @@
-Install and manage DCOS packages
+Description:
+    Install and manage software packages from the a DCOS package repository.
```
Comment (tamarrow) [inline cli/tests/data/help/package.txt]: @joel-hamill should we make this shorter to fit on a smaller screen?

**Pull request merged**